### PR TITLE
doc: releases: release-notes-3.7: Add info for A2DP and AVDTP

### PR DESCRIPTION
**Pull request by MarkWangChinese**
Added information about implementing A2DP and AVDTP to the Bluetooth section.

### CODE REVIEW

File: doc/releases/release-notes-3.7.rst [hunk 1]
```diff
@@ -116,6 +116,14 @@ Bluetooth
     Gateway is cellular phone. It controls the device (Hands-free Unit), that is the remote
     audio input and output mechanism.
 
+  * Implemented Advanced Audio Distribution Profile (A2DP) and Audio/Video Distribution Transport
+    Protocol (AVDTP), A2DP is enabled by the :kconfig:option:`CONFIG_BT_A2DP`, AVDTP is enabled
+    by the :kconfig:option:`CONFIG_BT_AVDTP`. They implement the protocols and procedures that
+    realize distribution of audio content of high quality in mono, stereo, or multi-channel modes.
+    A typical usage case is the streaming of music content from a stereo music player to headphones
```
Comment (aescolar) [inline doc/releases/release-notes-3.7.rst:123]:
```suggestion
    A typical use case is the streaming of music content from a stereo music player to headphones
```

File: doc/releases/release-notes-3.7.rst [hunk 2]
```diff
@@ -116,6 +116,14 @@ Bluetooth
     Gateway is cellular phone. It controls the device (Hands-free Unit), that is the remote
     audio input and output mechanism.
 
+  * Implemented Advanced Audio Distribution Profile (A2DP) and Audio/Video Distribution Transport
+    Protocol (AVDTP), A2DP is enabled by the :kconfig:option:`CONFIG_BT_A2DP`, AVDTP is enabled
+    by the :kconfig:option:`CONFIG_BT_AVDTP`. They implement the protocols and procedures that
```
Comment (aescolar) [inline doc/releases/release-notes-3.7.rst:121]:
```suggestion
    Protocol (AVDTP), A2DP is enabled by :kconfig:option:`CONFIG_BT_A2DP`, AVDTP is enabled
    by :kconfig:option:`CONFIG_BT_AVDTP`. They implement the protocols and procedures that
```

**Review by aescolar — COMMENTED** [review]
Thanks @MarkWangChinese , just 2 nits

File: doc/releases/release-notes-3.7.rst (hunk 1 above)
Comment (MarkWangChinese) [reply doc/releases/release-notes-3.7.rst]: Updated.

File: doc/releases/release-notes-3.7.rst (hunk 2 above)
Comment (MarkWangChinese) [reply doc/releases/release-notes-3.7.rst]: Updated.

**Review by aescolar — APPROVED** [review]

**Review by lylezhu2012 — APPROVED** [review]

**Review by kartben — APPROVED** [review]

**Review by dleach02 — APPROVED** [review]

**Pull request merged**